x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_actions/get_user_action_stats.ts

```diff
@@ -0,0 +1,207 @@
+        userActionTotals.total_comments + userActionTotals.total_other_actions
+      );
+    });
+
+    it('all "other" user action types are included', async () => {
+      // 1 creation action
+      const theCase = await createCase(supertest, postCaseReq);
+
+      // this update should account for 8 "other actions"
+      await updateCase({
+        supertest,
+        params: {
+          cases: [getCaseUpdateData(theCase.id, theCase.version)],
+        },
+      });
+
+      await bulkCreateAttachments({
+        supertest,
+        caseId: theCase.id,
+        params: [
+          // only this one should show up in total_comments
+          postCommentUserReq,
+          // the ones below count as total_other_actions
+          postExternalReferenceESReq,
+          persistableStateAttachment,
+          postCommentActionsReq,
+          postCommentAlertReq,
+        ],
+      });
+
+      const userActionTotals = await getCaseUserActionStats({ supertest, caseID: theCase.id });
+
+      expect(userActionTotals.total).to.equal(14);
+      expect(userActionTotals.total_comments).to.equal(1);
+      expect(userActionTotals.total_other_actions).to.equal(13);
+      expect(userActionTotals.total).to.equal(
+        userActionTotals.total_comments + userActionTotals.total_other_actions
+      );
+    });
+
+    describe('rbac', () => {
+      const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+      let theCase: CaseResponse;
+      beforeEach(async () => {
+        theCase = await createCase(supertestWithoutAuth, getPostCaseRequest(), 200, {
+          user: superUser,
+          space: 'space1',
+        });
+
+        await updateCase({
+          supertest: supertestWithoutAuth,
+          params: {
+            cases: [getCaseUpdateData(theCase.id, theCase.version)],
+          },
+          auth: superUserSpace1Auth,
+        });
+
+        await createComment({
+          supertest,
+          caseId: theCase.id,
+          params: postCommentUserReq,
+          auth: superUserSpace1Auth,
+        });
+      });
+
+      afterEach(async () => {
+        await deleteAllCaseItems(es);
+      });
+
+      it('should get the user actions for a case when the user has the correct permissions', async () => {
+        for (const user of [globalRead, superUser, secOnly, secOnlyRead, obsSec, obsSecRead]) {
+          const userActionTotals = await getCaseUserActionStats({
+            supertest: supertestWithoutAuth,
+            caseID: theCase.id,
+            auth: { user, space: 'space1' },
+          });
+
+          expect(userActionTotals.total).to.equal(10);
+          expect(userActionTotals.total_comments).to.equal(1);
+          expect(userActionTotals.total_other_actions).to.equal(9);
+          expect(userActionTotals.total).to.equal(
+            userActionTotals.total_comments + userActionTotals.total_other_actions
+          );
+        }
+      });
+
+      for (const scenario of [
```

Comment:
nit: Could you please add `obsOnly` and `obsOnlyRead` for `space1` in this scenario?